### PR TITLE
Add whisper-ctranslate2 flags to WhisperEngine.java

### PR DESCRIPTION
**Pull request by Arnei**
`whisper-ctranslate2`(https://github.com/Softcatala/whisper-ctranslate2) is an OpenAI whisper derivative with the main benefit of being significantly more performant on CPUs.
It also intentionally comes with the same CLI, so it works with our current implementation of whisper in `WhisperEngine.java`.

This PR makes mention of `whisper-ctranslate2` in the Opencast docs. It also adds configuration for two potentially useful flags exclusive to `whisper-ctranslate2` to `WhisperEngine.cfg`.

### Your pull request should…

* [x] have a concise title
* ~[ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists~
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by mliradelc** [comment]
I forgot completely about this PR. Codewise looks good. I only need to test with ctranslate2

**Comment by mwuttke** [comment]
Hello @mliradelc, have you had time to test whisper-ctranslate2?

**Comment by mliradelc** [comment]
I'm on it

**Comment by stefanosgeo** [comment]
@mliradelc @mtneug Can we ensure that this gets included in today's releases? We're scheduled to kick off with version 14.x in approx. 2 hours.

**Comment by mliradelc** [comment]
Yes, I'm making a couple of test, it should be no more than 30 minutes

File: etc/org.opencastproject.speechtotext.impl.engine.WhisperEngine.cfg [hunk 1]
```diff
@@ -6,3 +6,19 @@
...
+# Quantization is a technique that can reduce the model size and accelerate its execution with little to no degradation in accuracy.
+# On CPU int8 will give the best performance
+# Possible values: auto,int8,int8_float16,int16,float16,float32
+# Default: none
+#whisper.quantization: int8
+
+# Voice Activity Detection (VAD)
+# Filter out parts of the audio without speech using the Silero VAD model
+# Default: none
+#whisper.vad_enabled: true
```
Comment (mliradelc) [inline etc/org.opencastproject.speechtotext.impl.engine.WhisperEngine.cfg]: Need change ":" to "=" like in the suggestion.

```suggestion
# Default: none
#whisper.quantization=int8

# Voice Activity Detection (VAD)
# Filter out parts of the audio without speech using the Silero VAD model
# Default: none
#whisper.vad_enabled=true
```

**Pull request merged**